payment/tests/test_payment.py

```diff
@@ -169,22 +183,25 @@ def test_callbackpage_error_missing_argument(self, get_basket_mock):
         response = self.client.get(reverse('payment:error'), self.params)
         self.assertEqual(400, response.status_code)
 
+    @patch('payment.utils.register_user_verified_cohort')
     @patch('payment.utils.get_order')
-    def test_payment_notification_api(self, get_order_mock):
+    def test_payment_notification_api(self, get_order_mock, register_user_verified_cohort_mock):
         get_order_mock.return_value = self.api_response
-        data = {'username': self.user.username,
-                'email': self.user.email,
-                'order_number': '0'}
+        data = {
+            'username': self.user.username,
+            'email': self.user.email,
+            'order_number': '0'
+        }
 
-        self.client.post(reverse('fun-payment-api:payment-notification'),
-                json.dumps(data),
-                content_type="application/json")
-        self.assertEqual(1,
-                         SoftwareSecurePhotoVerification.objects.filter(
-                             user=self.user).count())
+        self.client.post(
+            reverse('fun-payment-api:payment-notification'),
+            json.dumps(data),
+            content_type="application/json"
+        )
+        self.assertEqual(1, SoftwareSecurePhotoVerification.objects.filter(user=self.user).count())
         self.assertEqual(1, len(mail.outbox))
-        self.assertEqual(True,
-                         mail.outbox[0].message().is_multipart())
+        self.assertTrue(mail.outbox[0].message().is_multipart())
+        register_user_verified_cohort_mock.assert_called_once()
```

Comment:
Why not validate call args while you're at it :wink: 
```suggestion
        register_user_verified_cohort_mock.assert_called_once_with(self.user, ???)
```